src/test/java/gov/usgs/cida/pubs/webservice/DeletedPublicationMvcServiceIT.java

```diff
@@ -38,13 +39,20 @@
 public class DeletedPublicationMvcServiceIT extends BaseIT {
 
 	@Autowired
+	private WebApplicationContext webApplicationContext;
+
 	private MockMvc mockMvc;
 
+	@Before
```

Comment:
These tests used to take advantage of Spring Boot test capabilities. Please go back to that rather than using straight Spring to set up the mockMvc.